static/docs/user-guide/install/linux.md

```diff
@@ -0,0 +1,104 @@
+# Installation on Linux
+
+## Install with pip
+
+```dvc
+$ pip install dvc
+```
+
+Depending on the type of the
+[remote storage](/doc/user-guide/external-dependencies) you plan to use, you
+might need to install optional dependencies: `[s3]`, `[ssh]`, `[gs]`, `[azure]`,
+and `[oss]`. Use `[all]` to include them all.
+
+<details>
+
+### Example: How to install DVC with support for Amazon S3 storage
+
+```dvc
+$ pip install 'dvc[s3]'
+```
+
+In this case it installs `boto3` library as well, besides DVC.
+
+</details>
+
+## Install with conda
+
+> Requires [Anaconda Distribution](https://docs.anaconda.com/anaconda/).
+
+```dvc
+$ conda install -c conda-forge dvc
+```
+
+> Currently, it supports only Python versions 2.7, 3.6, and 3.7.
+
+## Install from repo
+
+<details>
+
+### On Debian/Ubuntu
+
+```dvc
+$ sudo wget \
+       https://dvc.org/deb/dvc.list \
+       -O /etc/apt/sources.list.d/dvc.list
+
+$ sudo apt update
+
```

Comment:
should we remove these extra spaces? we don't put them in other places.

Reply:
It seemed toe me that it makes the content more readable. But I don't have a string opinion on this. Fixed it.